README.md

```diff
@@ -494,11 +488,7 @@ They pull a Docker image and then run a container with a Trino server:
   ```
 - Create release package and upload it to PyPI
   ```bash
-  . .venv/bin/activate &&
-  pip install twine &&
-  rm -rf dist/ &&
-  ./setup.py sdist bdist_wheel &&
-  twine upload dist/* &&
+  poetry publish --build &&
```

Comment:
any possibility of clearing old state/artifacts for safety?

Reply:
@hashhar I'm unsure. There's https://github.com/python-poetry/poetry/issues/1329 which has been open for some time.

Reply:
Let's keep the `rm -rf dist/` in the steps - I verified that Poetry indeed doesn't clean up old artifacts.